src/commands/config/ConfigAfkCheck.ts

```diff
@@ -594,6 +604,42 @@ export class ConfigAfkCheck extends BaseCommand {
                     newAfkCheckProps.prePostAfkCheckPermissions = p.value!;
                     break;
                 }
+                case "set_default_pos": {
+                    const n = await askInput<number>(
+                        ctx,
+                        botMsg,
+                        {
+                            embeds: [
+                                new MessageEmbed()
+                                    .setAuthor({name: ctx.guild!.name, iconURL: ctx.guild!.iconURL() ?? undefined})
+                                    .setTitle("Default AFK VC Position")
+                                    .setDescription(
+                                        "Here you can set the default raid AFK VC position. This is so that "
+                                        + "staff aren't confused by the position of the newly created channel at the top of "
+                                        + "the list. "
+                                        + "The current value is: " + StringUtil.codifyString(newAfkCheckProps.defaultPosition)
+                                        + "Type a __integer__ that represents the position of the channel. "
+                                        + "If you don't want to set this, press the **Back** button."
+                                    )
+                            ]
+                        },
+                        m => {
+                            const num = Number.parseInt(m.content, 10);
+                            return Number.isNaN(num) ? null : num;
+                        }
+                    );
+
+                    if (typeof n === "undefined") {
+                        await this.dispose(ctx, botMsg);
+                        return;
+                    }
+
+                    if (!n)
```

Comment:
If `n` is 0, then `!n` will return `false`. In other words, it appears that it's impossible to set `newAfkCheckProps.defaultPosition` to `0`.